### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by vincentchalamon**
Travis does not support PHP 8, I've migrated the configuration to GitHub Actions. Any help from @Nyholm to enable it in this repository would be appreciate :-)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      - master
+
+jobs:
+  tests:
+    name: Tests PHP ${{ matrix.php }} ${{ matrix.dependency }} (Symfony ${{ matrix.symfony }})
+    runs-on: ubuntu-latest
+    container:
+      image: php:${{ matrix.php }}
+      options: >-
+        --tmpfs /tmp:exec
```
Comment (Nyholm) [inline .github/workflows/ci.yml]: What is this for?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      matrix:
+        php:
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0.0RC2'
```
Comment (Nyholm) [inline .github/workflows/ci.yml]:
```suggestion
          - '8.0'
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Tests PHP ${{ matrix.php }} ${{ matrix.dependency }} (Symfony ${{ matrix.symfony }})
+    runs-on: ubuntu-latest
+    container:
+      image: php:${{ matrix.php }}
```
Comment (Nyholm) [inline .github/workflows/ci.yml]: Using docker packages like this seams a bit more complicated. What about just using `shivammathur/setup-php@2.7.0`?

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          - '8.0.0RC2'
+        symfony:
+          - '3.4.*'
+          - '4.4.*'
+          - '5.0.*'
+        include:
+          - php: '7.1'
+            coverage: true
+          - php: '7.1'
+            dependency: 'dunglas/symfony-lock:^3'
```
Comment (Nyholm) [inline .github/workflows/ci.yml]: We could stop using this package and use flex with `SYMFONY_REQUIRE: 3.4.*` instead.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+            dependency: 'dunglas/symfony-lock:^4'
+        prefer:
+          - ''
+          - 'lowest'
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: "Cache dependencies installed with composer"
```
Comment (Nyholm) [inline .github/workflows/ci.yml]: Cool!

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+        if: matrix.dependency != ''
+        run: composer require --no-update "${{ matrix.dependency }}"
+
+      - name: Update project dependencies
+        if: matrix.prefer == ''
+        run: composer update --no-progress --ansi --prefer-stable
+
+      - name: Update project dependencies lowest
+        if: matrix.prefer == 'lowest'
+        run: composer update --no-progress --ansi --prefer-stable --prefer-lowest
```
Comment (Nyholm) [inline .github/workflows/ci.yml]: This is not needed with `shivammathur/setup-php`

**Review by Nyholm — COMMENTED** [review]
Thank you for working on this

**Comment by vincentchalamon** [comment]
@Nyholm I think the GitHub Actions configuration must be in the repository to enable it

**Comment by Nyholm** [comment]
It is enabled. But there must be an existing workflow for it to run. 

I'll merge this PR and hope you will follow up on it to make sure it is green.

File: .github/workflows/ci.yml [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: curl, dom, iconv, intl, json, libxml, mbstring, phar, tokenizer, xdebug, xml, xmlwriter
```
Comment (Nyholm) [inline .github/workflows/ci.yml:45]: Lets use "flex" and "pcov". That is enough.

**Review by Nyholm — APPROVED** [review]

**Comment by vincentchalamon** [comment]
Thanks @Nyholm , I'm fixing it on #31 
By the way, would you like to enable this repository to the [Hacktoberfest](https://hacktoberfest.digitalocean.com/hacktoberfest-update)?

**Comment by Nyholm** [comment]
Thank you. 
I added the hackoctoberfest topic.

**Pull request merged**